### PR TITLE
feat: bump go parser v1.13.0

### PR DESCRIPTION
**Pull request by JamesPatrickGill**
This bumps the snyk-go-parser version to 1.13.0 - for more info see here https://github.com/snyk/snyk-go-parser/pull/45

### CODE REVIEW

**Review by admons — APPROVED** [review]

File: lib/index.ts [hunk 1]
```diff
@@ -697,3 +696,28 @@ function isBuiltinPackage(pkgName: string): boolean {
...
+    return { exactVersion, incompatible: !!incompatibleStr };
+  }
+}
+
+function toSnykVersion(v: ModuleVersion): string {
+  if ('hash' in v) {
+    return '#' + v['hash'];
+  } else {
+    return v.exactVersion.replace(/^v/, '');
+  }
```
Comment (darscan) [inline lib/index.ts:722]:
```suggestion
  if ('hash' in v && v.hash) {
    return '#' + v.hash;
  } else {
    return (v as { exactVersion: string }).exactVersion.replace(/^v/, '');
  }
```

Comment (darscan) [reply lib/index.ts:722]: maybe. ew

Comment (JamesPatrickGill) [reply lib/index.ts]: I took it

**Comment by darscan** [comment]
We'll need to land this first https://github.com/snyk/snyk-go-plugin/pull/115

**Pull request merged**